### ISSUE TITLE
Bulr everything (even on the objects not moving with respect to camera)

### ISSUE THREAD

**Issue by Manik2607** [Bauxitedev/godot-motion-blur#12]
a have a car following camera and car is getting blurred .
in a fps game gun is getting blurred.
in unreal engine it does not work like that.

it can be fixed by rendering objects separately but that is not ideal in most of the cases..

**Comment by Calinou** [comment]
This is noted in the limitations/known issues section of the README: https://github.com/Bauxitedev/godot-motion-blur#limitationsknown-issues

There is no way to fix this until Godot exposes access to motion vectors in shaders (which will only be done in 4.x, as Godot 3.x doesn't have motion vectors of any kind).

**Comment by Manik2607** [comment]
Is there a fixed port for godot ?

**Comment by Calinou** [comment]
> Is there a fixed port for godot ?

No, it's not feasible to do this without engine changes. Exposing motion vectors is technically feasible when TAA is enabled, but it's not trivial.
